2024-07-28 18:06:13,520 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-07-28 18:06:13,520 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\nn911\AppData\Local\Programs\Python\Python312\Lib\site-packages\instabot\bot\bot.py
2024-07-28 18:06:13,526 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-07-28 18:06:14,172 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2024-07-28 18:06:14,172 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2024-07-28 18:06:14,172 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2024-07-28 18:06:14,172 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
